contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -72,16 +72,16 @@ After loading the ClusterInfo from a file, the client MAY look for updated infor
 If the ClusterInfo information is hosted in a trusted place via HTTPS you can just request it that way.  This will use the root certificates that are installed on the system.  It may or may not be appropriate based on the user's constraints.  This method MUST use HTTPS.  Also, even though the payload for this URL is the `kubeconfig` format, it MUST NOT contain client credentials.
 
 ```
-kubeadm join --discovery-url="https://example/mycluster.yaml"
+kubeadm join --discovery-file="https://example/mycluster.yaml"
```

Comment:
Since it's utilizing the option to bring a file. Whether you read it from the internet using HTTPS or from disk doesn't _really_ matter. There were quite a long debate on this on the last edit to the proposal and on Slack I think, and we ended up with this option but the proposal wasn't reflected it seems.